From bcd1cd022c71921ce4b591cff248b42199bc16fd Mon Sep 17 00:00:00 2001
From: Meng Li <li.meng@amd.com>
Date: Tue, 15 Aug 2023 14:15:40 +0800
Subject: [PATCH 07/13] x86: Drop CPU_SUP_INTEL from SCHED_MC_PRIO for the
 expansion.

Drop CPU_SUP_INTE from SCHED_MC_PRIO, and it can be used by X86_AMD_PSTATE.

Signed-off-by: Meng Li <li.meng@amd.com>
---
 arch/x86/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index 5a9709cbd..a82f613f4 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -1059,8 +1059,9 @@ config SCHED_MC
 
 config SCHED_MC_PRIO
 	bool "CPU core priorities scheduler support"
-	depends on SCHED_MC && CPU_SUP_INTEL
+	depends on SCHED_MC
 	select X86_INTEL_PSTATE
+	select X86_AMD_PSTATE
 	select CPU_FREQ
 	default y
 	help
-- 
2.42.0.rc0.25.ga82fb66fed

